dts/bindings/dma/brcm,iproc-pax-dma.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Broadcom
+# SPDX-License-Identifier: Apache-2.0
+
+description: Broadcom iProc PAX(PCIE<->AXI) DMA controller
+
+include: dma-controller.yaml
+
+compatible: brcm,iproc-pax-dma
```

Comment:
Shouldn't you have different compatibles for v1/v2, given that they are programmed differently?

On that note: Is it possible to operate v2 in v1 mode, such that the v1 driver will work against that hardware? So should v2 be a more specific compatible, or are they mutually exclusive and need separate toplevel compatible?

Reply:
Different compatible is needed for  V1 and V2. Yes, they are programmed differently.

Both are mutually exclusive, not possible to operate V2 in V1 mode.

Reply:
Added two different compatible strings though dts entries perspective there's no distinct change required but driver point of view it's 2 different version of drivers.